templates/app-autoscaler-deployment.yml

```diff
@@ -196,6 +196,20 @@ instance_groups:
         agent:
           services:
             servicebroker: {}
+  - name: route_registrar
+    release: routing
+    consumes:
+      nats: {from: nats, deployment: cf}
+    properties:
+      route_registrar:
+        routes:
+        - name: autoscaler_service_broker
+          registration_interval: 20s
+          port: 6101
```

Comment:
don't hardcode port at multiple places, reference it from servicebroker properties.

Reply:
@rohitsharma04 how to reference from another job's properties as "properties defined in a Job block are accessible only to that job"?